docs/taskruns.md

```diff
@@ -96,6 +97,53 @@ spec:
           - --destination=gcr.io/my-project/gohelloworld
 ```
 
+#### Tekton Bundles
+
+You may also reference `Tasks` that are defined outside of your cluster using `Tekton
+Bundles`. A `Tekton Bundle` is an OCI artifact that contains Tekton resources like `Tasks`
+which can be referenced within a `taskRef`.
+
+```yaml
+spec:
+taskRef:
+  name: echo-task
+  bundle: docker.com/myrepo/mycatalog
+```
+
+Here, the `bundle` field is the full reference url to the artifact. The name is the
+`metadata.name` field of the `Task`. 
+
+You may also specify a `tag` as you would with a Docker image which will give you a fixed,
+repeatable reference to a `Task`.
```

Comment:
Wouldn't the digest be the "fixed repeatable reference"?

Reply:
AFAIK, best practice is to treat `tag`s as fixed elements but that is not an enforced aspect of the spec so I will call out digests as well